### PR TITLE
Add ns build

### PR DESCRIPTION
**Pull request by rcannood**
Adds a github action to build a namespace from many viash config files.

### CODE REVIEW

File: ns-build/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+ns-build
```
Comment (DriesSchaumont) [inline ns-build/README.md:1]: Could you add a CHANGELOG entry?

File: ns-build/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- `query`: - *optional*. Filter which components get selected by
+  component and namespace name. Can be a regex. Example:
+  “^mynamespace/component1\$”.
+- `query_namespace`: - *optional*. Filter which namespaces get selected
+  by namespace name. Can be a regex. Example: “^mynamespace\$”.
+- `query_name`: - *optional*. Filter which components get selected by
+  component name. Can be a regex. Example: “^component1”.
+- `src`: - *optional*. A source directory containing viash config files,
+  possibly structured in a hierarchical folder structure. Default: src/.
+- `platform`: - *optional*. Acts as a regular expression to filter the
```
Comment (DriesSchaumont) [inline ns-build/README.md:23]: Maybe add options here? Although it is a pain when we add a new platform. Could be a link to the viash documentation for the platforms... But this link might also change...
Or just leave as is 😄

File: ns-build/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Build a namespace from many viash config files.
+
+## Usage
+
+This action will run `viash ns build`.
+
+We recommend using a Linux or MacOS runner if possible.
+
+### Inputs available
```
Comment (DriesSchaumont) [inline ns-build/README.md:12]: Perhaps add a general comment to look at the documentation for `viash ns build` on the website to get a more in-depth look at the command?

**Review by DriesSchaumont — APPROVED** [review]
LGTM! Some small suggestions, but those are not blocking for merging so feel free to merge.

**Review by DriesSchaumont — CHANGES_REQUESTED** [review]
Never mind, forgot about the CHANGELOG entry

File: ns-build/README.md (hunk 2 above)
Comment (rcannood) [reply ns-build/README.md:25]: Oh you mean the different platforms. Eh. I'd leave it up to the user to know what they're doing, at least for now.

File: ns-build/README.md (hunk 1 above)
Comment (rcannood) [reply ns-build/README.md:1]: Solved!

File: ns-build/README.md (hunk 3 above)
Comment (rcannood) [reply ns-build/README.md:14]: Solved!

**Review by DriesSchaumont — APPROVED** [review]

**Pull request merged**